cudf/Intro to cuDF.ipynb

```diff
@@ -814,122 +979,129 @@
       "1    2\n",
       "2    3\n",
       "3     \n",
-      "4    4\n",
+      "4    5\n",
       "0    1\n",
       "1    2\n",
       "2    3\n",
       "3     \n",
-      "4    4\n",
-      "dtype: int64\n",
-      "   a   b  c\n",
-      "0  0  19  0\n",
-      "1  1  18  1\n",
-      "2  2  17  2\n",
-      "3  3  16  3\n",
-      "4  4  15  4\n",
-      "5  0  19  0\n",
-      "6  1  18  1\n",
-      "7  2  17  2\n",
-      "8  3  16  3\n",
-      "9  4  15  4\n"
+      "4    5\n",
+      "dtype: int64\n"
      ]
     }
    ],
    "source": [
-    "print(cudf.concat([s, s]))\n",
-    "print(cudf.concat([df.head(), df.head()], ignore_index=True))"
+    "s = cudf.Series([1, 2, 3, None, 5])\n",
+    "print(cudf.concat([s, s]))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Join"
+    "## Append"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Performing SQL style merges."
+    "Appending values from another `Series` or array-like object."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 19,
-   "metadata": {
-    "collapsed": false
-   },
+   "execution_count": 215,
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "   key  vals_a  vals_b\n",
-      "3    0    10.0        \n",
-      "0    1    11.0    10.0\n",
-      "1    2    12.0    11.0\n",
-      "4    3    13.0        \n",
-      "2    4    14.0    12.0\n"
+      "0    1\n",
+      "1    2\n",
+      "2    3\n",
+      "3     \n",
+      "4    5\n",
+      "0    1\n",
+      "1    2\n",
+      "2    3\n",
+      "3     \n",
+      "4    5\n",
+      "dtype: int64\n"
      ]
     }
    ],
    "source": [
-    "df_a = cudf.DataFrame()\n",
-    "df_a['key'] = [0, 1, 2, 3, 4]\n",
-    "df_a['vals_a'] = [float(i + 10) for i in range(5)]\n",
-    "\n",
-    "df_b = cudf.DataFrame()\n",
-    "df_b['key'] = [1, 2, 4]\n",
-    "df_b['vals_b'] = [float(i+10) for i in range(3)]\n",
-    "\n",
-    "df_merged = df_a.merge(df_b, on=['key'], how='left')\n",
-    "print(df_merged.sort_values('key'))"
+    "print(s.append(s))"
```

Comment:
This fails for me. I guess it's because I'm using cudf 0.7?

Reply:
Ah, didn't think about that. It's due to the nulls. The fix for this is done, but it won't be "officially released" until 0.8

Reply:
We'll go with 0.8 based on slack discussion